src/components/molecules/TrackCard/index.tsx

```diff
@@ -0,0 +1,80 @@
+import { Typography, Box } from '@mui/material'
+import { Colors } from 'src/styles/color'
+import Image, { StaticImageData } from 'next/image'
+
+export interface TrackCardProps {
+  id?: string
+  title: string
+  minute?: number
+  speaker?: string
+  speakerIcon?: string | StaticImageData
+  color?: string
+  idColor: string
+  fullWidth: boolean
+}
+
+export const TrackCard = ({
+  id,
+  title,
+  minute,
+  speaker,
+  speakerIcon,
+  color = Colors.background.secondary_pink,
+  idColor = Colors.background.primary_pink,
+  fullWidth = false
+}: TrackCardProps) => {
+  return (
+    <Box position="relative">
+      <Box
+        sx={{
+          backgroundColor: color,
+          // TODO(@maito1201): fullWidthがtrueの時の挙動はタイムテーブルの時刻表示を考慮したものを実装する
+          width: fullWidth ? '100vw' : '100%',
+          padding: '16px',
+          position: 'absolute',
+          borderRadius: '4px'
+        }}
+      >
+        <Box
+          sx={{
+            display: 'flex',
+            backgroundColor: idColor,
+            borderRadius: '2px',
+            width: 'fit-content',
+            padding: '0 4px'
+          }}
+        >
```

Comment:
ID と タイトルの間の余白 4px が取れてなさそうです。

```suggestion
        <Box
          sx={{
            display: 'flex',
            backgroundColor: idColor,
            borderRadius: '2px',
            width: 'fit-content',
            padding: '0 4px',
            mb: '4px'
          }}
        >
```